simple_history/locale/fr/LC_MESSAGES/django.po

```diff
@@ -7,132 +7,132 @@ msgid ""
 msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-04-25 22:00+0200\n"
+"POT-Creation-Date: 2021-04-04 01:50+0300\n"
 "PO-Revision-Date: 2020-04-25 22:50+0200\n"
+"Last-Translator: \n"
+"Language-Team: \n"
 "Language: fr\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n > 1);\n"
-"Last-Translator: \n"
-"Language-Team: \n"
 "X-Generator: Poedit 2.3\n"
 
-#: simple_history/admin.py:111
+#: .\simple_history\admin.py:102
```

Comment:
Why are all of the slashes now backslashes?

Reply:
Because sadly, path encoding depends on the platform `gettext` is running on.
I'm not sure whether this is a big deal or not, as running `makemessages` again on a GNU/Linux system would revert to `/`.

Reply:
Would it be worth overriding the `makemessages` command to normalize the paths? Not sure why Django doesn’t do this by default (AFAIK, Unix-style paths work fine on Windows).

Reply:
I'm not sure how 3rd-party translation tools manage with this.
Overriding `makemessages` might be an idea, but it doesn't sound very project-specific to me. I guess Django preserves default `gettext` behavior, as generally you'll run `makemessages` on the platform you are translating on before making any edits.

Reply:
(in the context of this somewhat trivial PR, I'll rather do a quick regex replace and not dive into a `makemessages` override and the tests that it would need)